packages/story-editor/src/components/panels/shared/media/backgroundAudioPanelContent.js

```diff
@@ -235,73 +265,72 @@ function BackgroundAudioPanelContent({
             />
           )}
           {enableAudioHotlinking && (
-            <>
-              <UploadButton
-                variant={BUTTON_VARIANTS.RECTANGLE}
-                type={BUTTON_TYPES.SECONDARY}
-                size={BUTTON_SIZES.SMALL}
-                onClick={() => setIsOpen(true)}
-              >
-                {__('Link to audio file', 'web-stories')}
-              </UploadButton>
-              <HotlinkModal
-                title={__('Insert external background audio', 'web-stories')}
-                isOpen={isOpen}
-                onSelect={onSelectHotlink}
-                onClose={() => setIsOpen(false)}
-                allowedFileTypes={allowedAudioFileTypes}
-                insertText={__('Use audio file', 'web-stories')}
-                insertingText={__('Selecting audio file', 'web-stories')}
-              />
-            </>
+            <UploadButton
+              variant={BUTTON_VARIANTS.RECTANGLE}
+              type={BUTTON_TYPES.SECONDARY}
+              size={BUTTON_SIZES.SMALL}
+              onClick={() => setIsHotlinkDialogOpen(true)}
+            >
+              {__('Link to audio file', 'web-stories')}
+            </UploadButton>
           )}
         </ButtonRow>
       )}
       {resource?.src && (
         <>
-          <Row>
+          <FileRow
+            id={resource.id}
+            src={resource.src}
+            title={resource.src.split('/').pop()}
```

Comment:
The title should get truncated just in case.

<img width="396" alt="Screen Shot 2022-05-20 at 5 49 39 AM" src="https://user-images.githubusercontent.com/62242/169502862-3e5cde11-fdd2-4b52-9408-67f2d7aed21e.png">

Reply:
Ah good point. Will add some CSS for that 👍